inkscape/src/main/assembly/dist/lace_ground.inx

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <inkscape-extension xmlns="http://www.inkscape.org/namespace/inkscape/extension">
 
-	<_name>Lace Ground</_name>
+	<_name>Ground from Template</_name>

```

Comment:
Take a look at the [second tab](https://cdn.rawgit.com/jo-pol/DiBL/4209f63/standalone/tiles/dibl-tiles/src/main/assembly/web/tabs.html#tab2) you can generate both (color coded) pair diagrams as thread diagrams. The fun of this choice is you can generate matching diagrams. Implementation of more stitches is waiting for #1 

At the beginning you minted pair traversals, they have only topology, my diagrams add geometry and thus become working diagrams. But topology and geometry is mathematicians speach, not lacemakers speach.